.ci/jenkins/Jenkinsfile.prod.nightly

```
@@ -170,13 +174,18 @@ pipeline {
     }
 }
 
-def getMessageBody(String propertiesFileUrl, String alreadyBuiltProjects) { 
-    def alreadyBuiltProjectsArray = (alreadyBuiltProjects ?: '').split(";")
+/**
+ * 
+ * @param propertiesFileUrl the propertiesFile URL
+ * @param pmeExecutionResult the PME execution map [projectName: projectVersion]
+ * @return the UMB message body composed the propertiesFileUrl and the builtProjects containing the projectName: projectVersion map 
+ */
+def getMessageBody(String propertiesFileUrl, Map pmeExecutionResult) {
     return """
 {
 	"propertiesFileUrl": "${propertiesFileUrl}",
-	"builtProjects": ${new groovy.json.JsonBuilder(alreadyBuiltProjectsArray).toString()}
-}"""    
+	"builtProjects": ${new groovy.json.JsonBuilder(pmeExecutionResult).toString()}
```

The project versions are already into the properties file, so I don't see the point of sending them in the built projects ...

```suggestion
	"builtProjects": ${new groovy.json.JsonBuilder(pmeExecutionResult.collect{ it.key }).toString()}
```
would keep the backward compatibility